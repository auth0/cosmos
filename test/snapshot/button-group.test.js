import React from 'react'
import renderer from 'react-test-renderer'
import { ButtonGroup } from '@auth0/cosmos'
import 'jest-styled-components'

describe('ButtonGroup', () => {
  it('renders with automation attribute', () => {
    const tree = renderer.create(<ButtonGroup />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
