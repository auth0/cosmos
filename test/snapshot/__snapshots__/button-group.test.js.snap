// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup renders with automation attribute 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
}

.c0 .cosmos-tooltip {
  left: 40%;
}

.c0 .sc-gqjmRU {
  margin-right: 8px;
}

<div
  className="c0"
  data-cosmos-key="button-group"
/>
`;
